socorro/database/database.py

```diff
@@ -1,200 +0,0 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
There's no evidence of `from socorro.database` or `from socorro import database` (code or configuration) except for uses of `transaction_executor` which is used in a lot of places, so I removed this file.